tests/pom.xml

```diff
@@ -16,7 +16,7 @@
     <!--
     These properties are read in by arquillian.xml
     -->
-    <browser>phantomjs</browser> <!-- PhantomJS will be our default browser if no profile is specified-->
+    <browser>htmlUnit</browser> <!-- htmlUnit will be our default browser if no profile is specified-->
```

Comment:
I was getting better debugging output from htmlUnit, but it does not really matter.